.github/workflows/deploy.yml

```diff
@@ -3,14 +3,14 @@ name: Deploy
 on:
   release:
     types: [created]
-   
+
 
 jobs:
   deploy:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9]
+        python-version: [3.11]
```

Comment:
Not  sure if we need to specify python-version here for the deploy workflow. would either go with 3.12 or use the ubuntu-latest default?

Reply:
yes, we need that. the python version is later referenced